docs/cudf/source/user_guide/io/read-json.md

```diff
@@ -19,14 +19,14 @@ each object corresponds to a row.
     {"a": "v2", "b": 7},
     {"a": "v3", "b": 5}
 ]'''
->>> df_records = cudf.read_json(j, engine='cudf_experimental')
+>>> df_records = cudf.read_json(j, engine='cudf')
```

Comment:
Should We remove the engine parameter when the default value is passed?

Reply:
Dropped in docs 👍